llvm/lib/Target/AMDGPU/VOPInstructions.td

```diff
@@ -1342,17 +1500,43 @@ class Base_VOP3_DPP16<bits<10> op, VOP_DPP_Pseudo ps, string opName = ps.OpName>
   let OtherPredicates = ps.OtherPredicates;
 }
 
+class Base_VOP3_DPP16_t16<bits<10> op, VOP_DPP_Pseudo ps,
```

Comment:
Can you replace the word Base in this // VOP3 DPP section to be another word? Maybe 'Intermediate'? This will allow us to follow the new convention at L784 where Base means the class is shared between T16 and regular.

Reply:
Maybe 'Helper' I like Helper better than Type or Class.

Reply:
done